cookbooks/app-react-router/src/pages/index.ts

```diff
@@ -0,0 +1,11 @@
+import { index, route, prefix } from '@equinor/fusion-framework-react-router/routes';
+
+export const pages = [
+  index('./HomePage.tsx'),
+  prefix('products', [index('./ProductsPage.tsx'), route(':id', './ProductPage.tsx')]),
+  prefix('users', [index('./UsersPage.tsx'), route(':id', './UserPage.tsx')]),
```

Comment:
format array on new lines for more readability

Reply:
sorry mac, thats biome